tox.ini

```diff
@@ -113,7 +119,9 @@ commands =
 # integration tests
     {py27,py34}-integration: nosetests {posargs} tests/test_integration.py
 # run all tests for the release jobs except the ones with a different test runner
-    {py27,py34}-contrib: nosetests {posargs} --exclude=".*(django).*" tests/contrib/
+    {py27,py34}-contrib: nosetests {posargs} --exclude=".*(django|asyncio|aiohttp).*" tests/contrib/
+    {py35,py36}-async: nosetests {posargs} tests/contrib/asyncio
```

Comment:
fixed in https://github.com/DataDog/dd-trace-py/pull/192 and https://github.com/DataDog/dd-trace-py/pull/191